mail-storage/src/main/java/mail/storage/dto/UpdateMessageDto.java

```diff
@@ -0,0 +1,26 @@
+package mail.storage.dto;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotEmpty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import mail.storage.domain.MessageType;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class UpdateMessageDto {
```

Comment:
why do you need a separate Dto for update?

Reply:
The UpdateMessageDto differs from a basic Dto because it contains fields that are suitable for certain business logic for update. If I needed to delete this class, I would use the MessageDto which has excessive information that is not needed during update.